[0m21:09:38.340887 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025BA8E5DAF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025BAC70BD90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025BAC70BB80>]}


============================== 21:09:38.361344 | 02d682a9-ed6b-4e29-abb0-b684fefe4788 ==============================
[0m21:09:38.361344 [info ] [MainThread]: Running with dbt=1.8.0
[0m21:09:38.363298 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\97450\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init google_analytics', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m21:09:38.640477 [debug] [MainThread]: Starter project path: D:\Github Repositories\myenv\lib\site-packages\dbt\include\starter_project
[0m21:09:38.687286 [info ] [MainThread]: 
Your new dbt project "google_analytics" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m21:09:38.689239 [info ] [MainThread]: Setting up your profile.
[0m21:15:14.105657 [info ] [MainThread]: Profile google_analytics written to C:\Users\97450\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m21:15:14.109563 [debug] [MainThread]: Command `dbt init` succeeded at 21:15:14.109563 after 336.01 seconds
[0m21:15:14.111516 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025BA8E5DAF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025BAC70BB20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025BAC6F6FA0>]}
[0m21:15:14.113465 [debug] [MainThread]: Flushing usage events
[0m21:15:43.045864 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FF9E7D8BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFA108BDC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFA108BBB0>]}


============================== 21:15:43.054635 | 07c54a30-84ef-4ea3-8224-5f198e38ae1e ==============================
[0m21:15:43.054635 [info ] [MainThread]: Running with dbt=1.8.0
[0m21:15:43.056549 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\97450\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m21:15:43.134364 [info ] [MainThread]: dbt version: 1.8.0
[0m21:15:43.136316 [info ] [MainThread]: python version: 3.9.13
[0m21:15:43.137328 [info ] [MainThread]: python path: D:\Github Repositories\myenv\Scripts\python.exe
[0m21:15:43.139282 [info ] [MainThread]: os info: Windows-10-10.0.22631-SP0
[0m21:15:45.562496 [info ] [MainThread]: Using profiles dir at C:\Users\97450\.dbt
[0m21:15:45.564492 [info ] [MainThread]: Using profiles.yml file at C:\Users\97450\.dbt\profiles.yml
[0m21:15:45.566400 [info ] [MainThread]: Using dbt_project.yml file at D:\Github Repositories\dbt-projects\google-analytics\dbt_project.yml
[0m21:15:45.567413 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m21:15:45.571314 [debug] [MainThread]: Command `dbt debug` failed at 21:15:45.570307 after 2.70 seconds
[0m21:15:45.572255 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FF9E7D8BB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFB1F17190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFA108BBE0>]}
[0m21:15:45.573269 [debug] [MainThread]: Flushing usage events
[0m21:18:06.572500 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFE409DB20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFE694FD30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFE694FB20>]}


============================== 21:18:06.581325 | 789e033e-0973-487f-9aa2-60ee33da76b2 ==============================
[0m21:18:06.581325 [info ] [MainThread]: Running with dbt=1.8.0
[0m21:18:06.583273 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\97450\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt debug', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m21:18:06.632651 [info ] [MainThread]: dbt version: 1.8.0
[0m21:18:06.633632 [info ] [MainThread]: python version: 3.9.13
[0m21:18:06.635544 [info ] [MainThread]: python path: D:\Github Repositories\myenv\Scripts\python.exe
[0m21:18:06.637497 [info ] [MainThread]: os info: Windows-10-10.0.22631-SP0
[0m21:18:08.975705 [info ] [MainThread]: Using profiles dir at C:\Users\97450\.dbt
[0m21:18:08.977659 [info ] [MainThread]: Using profiles.yml file at C:\Users\97450\.dbt\profiles.yml
[0m21:18:08.979574 [info ] [MainThread]: Using dbt_project.yml file at D:\Github Repositories\dbt-projects\google-analytics\dbt_project.yml
[0m21:18:08.981523 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m21:18:08.984451 [debug] [MainThread]: Command `dbt debug` failed at 21:18:08.984451 after 2.58 seconds
[0m21:18:08.985427 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFE409DB20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFE69EC0D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001FFE694FB50>]}
[0m21:18:08.987422 [debug] [MainThread]: Flushing usage events
[0m21:21:30.043780 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027B1741DB80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027B19CA1160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027B19CA1820>]}


============================== 21:21:30.052521 | 7edfc74f-bf6a-4934-bcd0-bc83fc55e64b ==============================
[0m21:21:30.052521 [info ] [MainThread]: Running with dbt=1.8.0
[0m21:21:30.054510 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\97450\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m21:21:30.056468 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\Github Repositories\dbt-projects\google-analytics\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:21:30.059353 [debug] [MainThread]: Command `dbt run` failed at 21:21:30.059353 after 0.18 seconds
[0m21:21:30.061305 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027B1741DB80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027B19CA1790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027B19CA1520>]}
[0m21:21:30.062281 [debug] [MainThread]: Flushing usage events
